### PR TITLE
chore: add observibility

### PR DESCRIPTION
**Pull request by shanshanying**
update readme with podMonitor

### CODE REVIEW

**Review by shanshanying — COMMENTED** [review]
/approve

**Review by wangyelei — APPROVED** [review]

File: examples/postgresql/README.md [hunk 1]
```diff
@@ -499,26 +575,19 @@ kubectl apply -f examples/postgresql/upgrade.yaml
...
-```bash
-kubectl edit cluster pg-cluster
-```
+Alternatively, you may modify the `spec.componentSpecs.serviceVersion` field to the desired version to upgrade the cluster.
 
-And modify the `spec.componentSpecs.serviceVersion` field to the desired version.
-
-You are suggested to check the compatibility of versions before upgrading, using command:
+> [!Warning]
+> Do rembmer to to check the compatibility of versions before upgrading the cluster.
```
Comment (Y-Rookie) [inline examples/postgresql/README.md:585]: rembmer -> remember

**Review by Y-Rookie — APPROVED** [review]

File: examples/postgresql/README.md (hunk 1 above)
Comment (shanshanying) [reply examples/postgresql/README.md]: fixed

**Review by Y-Rookie — APPROVED** [review]

**Review by wangyelei — APPROVED** [review]

**Pull request merged**